packages/metro-config/src/defaults/index.js

```diff
@@ -111,6 +111,7 @@ const getDefaultValues = (projectRoot: ?string): ConfigT => ({
       },
       output: {
         ascii_only: true,
+        comments: false,
```

Comment:
1. I don't see a need to specify this explicitly.
2. Our default minifier, Terser, has a [documented](https://terser.org/docs/api-reference#format-options) default of `"some"`, set [here](https://github.com/terser/terser/blob/e072c218498ff5bcacab268f8813d5480ba0d288/lib/cli.js#L102), so this overrides that and may be an unwanted change in behaviour. Per 1, let's just remove this explicit Metro default.

```suggestion
```

Reply:
Previously, we had `comments: false` in Babel, which basically omits all comments, so no comments ever reach the minifier. To preserve that behaviour, we need to set `comments: false` here. If you still want to remove it, that's fine with me, but it will produce a different output than today.

Reply:
I see! Yeah, in that case we should probably land this with a behaviour-preserving default so it can go out as a patch, then align with terser's default in the next major. (Would you mind sending a separate PR with that?) 

The only thing I don't get is: if this is supposed to be behaviour-preserving, why did the buildGraph test snapshot change? Do we have a good test covering the actual default from Metro's config?

Reply:
It changed because it's unminified, so the comments have yet to be stripped out.

I can send out a separate PR to remove this line.

Reply:
Thanks for merging this PR! Here's the follow-up: https://github.com/facebook/metro/pull/972